xcdat/spatial.py

```diff
@@ -359,7 +361,7 @@ def _validate_region_bounds(self, axis: SpatialAxis, bounds: RegionAxisBounds):
                 "upper bounds, (lower, upper)."
             )
 
-        if len(bounds) <= 0 or len(bounds) > 2:
+        if len(bounds) != 2:
```

Comment:
I randomly caught this... I think `<=0` doesn't really make any sense, right?

Reply:
I agree with this change, and `<=0` does not make sense (arrays can't have < 0 elements)